src/Entities/Schema.php

```diff
@@ -165,6 +165,9 @@ public function __construct()
             'transactions/{transactionId}' => function (array $content) {
                 return new Transaction($content);
             },
+            'payouts' => function (array $content) {
+                return new Payout($content);
+            },
```

Comment:
```suggestion
            'payouts' => function (array $content) {
                return new Payout($content);
            },
```

There is no responses with such resource, as far I understand the API definition.

Reply:
You're right :+1: